sdk/storage/azure-storage-files-shares/CHANGELOG.md

```diff
@@ -10,6 +10,14 @@
 
 - Changed the return type of `StartCopy` API from a `Response<T>` to the particular `Operation<T>` type called `StartCopyShareFileOperation` directly.
 - String conversion functions of extensible enums were renamed from `Get()` to `ToString()`.
+- Renamed `GetSharePropertiesResult` to `ShareProperties`.
+- Renamed `GetShareStatisticsResult` to `ShareStatistics`.
+- Renamed `GetShareDirectoryPropertiesResult` to `ShareDirectoryProperties`.
+- Renamed `GetShareFilePropertiesResult` to `ShareFileProperties`.
+- Renamed `GetShareFileRangeListResult` to `ShareFileRangeInfo`.
+- `ShareClient::GetPermission` now returns `Azure::Response<std::string>`.
+- `ShareClient::GetAccessPolicy` now returns `Azure::Response<std::vector<Models::SignedIdentifier>>`.
```

Comment:
Since this operation returns important response headers and a body, keep the wrapping struct called `AccessPolicyResult`.